### PR TITLE
upgrade base image

### PR DESCRIPTION
**Pull request by ameukam**
Upgrade base image to 20.04

Signed-off-by: Arnaud Meukam <ameukam@gmail.com>

### CODE REVIEW

**Comment by ameukam** [comment]
Also tests will be skipped for 1.12-1.14: https://github.com/kubernetes/test-infra/pull/17443

**Comment by aojea** [comment]
/hold

let's wait for CNI 0.8.6, I was told they are going to release it this week, and it has a change in the portmap plugin I'm interested in see how it goes

can you add this change @ameukam 
CNI_VERSION="v0.8.3" -> v0.8.6

I'll ping you once it has been released

**Comment by BenTheElder** [comment]
I'd just as soon upgrade CNI seperately, easier to track down issues :-)

didn't touch this prior to finishing 0.8, but that's done now so let's go :-)

**Comment by BenTheElder** [comment]
rebased, pushing an image

**Comment by BenTheElder** [comment]
added crictl v1.18.0

**Comment by BenTheElder** [comment]
/hold cancel
/lgtm
/approve
thanks!

File: images/base/Dockerfile [hunk 1]
```diff
@@ -19,15 +19,15 @@
...
 # start from ubuntu 19.10, this image is reasonably small as a starting point
 # for a kubernetes node image, it doesn't contain much we don't need
-FROM ubuntu:19.10
+FROM ubuntu:20.04
 
 # Configure containerd and runc binaries from kind-ci/containerd-nightlies repository
 # The repository contains latest stable releases and nightlies built for multiple architectures
 ARG CONTAINERD_VERSION="v1.3.3-14-g449e9269"
 # Configure CNI binaries from upstream
 ARG CNI_VERSION="v0.8.5"
```
Comment (BenTheElder) [inline images/base/Dockerfile:28]: we'll come back for this when it's out

Comment (aojea) [reply images/base/Dockerfile:28]: no probl, I  was expecting a release this week, but it wasn't the case obviously :)

**Pull request merged**